src/components/index.ts

```diff
@@ -0,0 +1,8 @@
+export type { IContextTypes, IProviderTypes } from "../Context/contextTypes"
```

Comment:
It's better not to collect all types in one file. It can lead to circular dependency on one hand and possible problems in navigating from another. It's better to keep types as close as possible to file when they are used.